src/core/server/plugins/types.ts

```diff
@@ -204,12 +205,14 @@ export const SharedGlobalConfigKeys = {
   kibana: ['defaultAppId', 'index'] as const,
   elasticsearch: ['shardTimeout', 'requestTimeout', 'pingTimeout', 'startupTimeout'] as const,
   path: ['data'] as const,
+  csp: ['rules', 'strict', 'warnLegacyBrowsers'] as const,
```

Comment:
Why we expose all csp values? As I can see, plugins use only `csp.rules`. Other settings are platform-specific.